builders/tools/walletconnect.md

```diff
@@ -0,0 +1,416 @@
+---
+
+# Integrate WalletConnect into a DApp
+
+![WalletConnect Banner](/images/builders/tools/walletconnect/walletconnect-banner.png)
+
+## Introduction {: #introduction } 
+
+[WalletConnect](https://walletconnect.com/) is an open protocol to communicate securely between wallets and DApps. 
+
+WalletConnect establishes a remote connection between a DApp and mobile wallet by using a bridge server to relay payloads. The connection is initiated via a QR code displayed in a DApp, which will need to be scanned and approved by a mobile wallet. When a connection is established, the payloads between the DApp and wallet are encrypted through a shared key.
+
+![WalletConnect flow](/images/builders/tools/walletconnect/walletconnect-1.png)
+
+WalletConnect can also be used to establish a connection between a DApp and a desktop wallet. However, this guide will only cover a mobile wallet connection.
+
+In this guide, you'll learn how to integrate WalletConnect into a simple DApp built on the Moonbase Alpha TestNet. The guide will be divided into a few different sections. The first section will cover connecting your DApp to MetaMask mobile. After the connection has been established, the guide will cover disconnections. This way, when you are testing your DApp you will be able to connect and then disconnect so you don't end up with a bunch of unnecessary WalletConnect sessions lingering in MetaMask mobile. Afterwards you will learn how to display network and account details when connected, and send transactions from your DApp to MetaMask mobile for confirmation.
+
+This guide is an adaptation of the [WalletConnect Example Dapp](https://example.walletconnect.org/){target=blank} ([source code](https://github.com/WalletConnect/walletconnect-example-dapp){target=blank}). To view the end result, you can check out the [Moonbeam WalletConnect Demo app](https://moonbeam-walletconnect-demo.netlify.app/){target=blank} ([source code](https://github.com/PureStake/moonbeam-walletconnect-demo){target=blank}).
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+Throughout this guide, you'll use a simple front-end DApp built with [React](https://reactjs.org/) to connect to a mobile wallet via WalletConnect. So you will need a React project and the MetaMask mobile app installed for testing purposes. A template has been created that includes the required packages, some basic styling, and placeholders where logic and UI elements will need to be added. However, if you would to use your own DApp, you'll need to install the following required dependencies:
+
+```
+npm install ethers @walletconnect/client @walletconnect/qrcode-modal
+```
+
+This guide will use MetaMask mobile for testing purposes. You can install MetaMask mobile from the [Apple App Store](https://apps.apple.com/us/app/metamask-blockchain-wallet/id1438144202) or [Google Play Store](https://play.google.com/store/apps/details?id=io.metamask&hl=en_US&gl=US).
```

Comment:
It is not good practice to include links to the play store (for example, if they change it in the future). I think you can just link to Metamask website